platform/ios/app/MBXViewController.m

```diff
@@ -165,7 +165,9 @@ - (void)viewDidLoad
     self.mapView.scaleBar.hidden = NO;
     self.mapView.showsUserHeadingIndicator = YES;
     self.hudLabel.hidden = YES;
-    self.hudLabel.titleLabel.font = [UIFont monospacedDigitSystemFontOfSize:10 weight:UIFontWeightRegular];
+    if ([UIFont respondsToSelector:@selector(monospacedDigitSystemFontOfSize:weight:)]) {
```

Comment:
For iOS 8 which would be the font alternative?

Reply:
It’s set to use the system font of the same size, via the storyboard.